### DISCUSSION TITLE
Why was pycord moved to a different documentation website?

### DISCUSSION THREAD

**Discussion by brdcat-dev** [Pycord-Development/pycord#2270] [Q&A]
Not sure exactly when this happened, but the move to docs.pycord.dev, instead of readthedocs is incredibly disappointing, The API reference especially is missing some key info and feels a lot more difficult to navigate. I've noticed properties of a class, i.e `Guild` like `Guild.channels` don't show that is a `list[abc.GuildChannel]` type. This made navigating incredibly easy and fast now its gone :(

**Comment by Lulalaby** [comment]
docs.pycord.dev is readthedocs but with a custom domain.
Nothing changed in that regards. Only how it's split is changed.

**Reply by brdcat-dev** [reply]
Sorry then, I mean't to compare it to the [old discordpy](https://discordpy.readthedocs.io/en/stable/api.html#discord.TextChannel.history) repository and how they documented it, it seems a lot easier than the current docs.

**Reply by Lulalaby** [reply]
We've decided for the new layout to make it easier to read and maintain.

**Reply by brdcat-dev** [reply]
Is there anyway to access the last version of that documentation?

**Reply by Lulalaby** [reply]
The docs for our main branch are here: https://docs.pycord.dev/en/master/api/models.html

We won't be changing back tho

**Reply by brdcat-dev** [reply]
Thats a shame :(, anyway thanks for the clarification!

**Reply by JustaSqu1d** [reply]
There is an issue opened regarding this (#2141) and we plan to have it changed.

**Reply by Lulalaby** [reply]
forgot about it. thanks squid. can you work on it :p

**Reply by JustaSqu1d** [reply]
I don't know how to, but I can try doing that this weekend.

**Reply by Lulalaby** [reply]
appreciate it
